hub/codec/tests/test_codecs.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+import numpy as np
+
+from hub.codec import PngCodec, NumPy, Lz4, Zstd, JpegCodec
+
+
+@pytest.mark.parametrize("from_config", [False, True])
+def test_png_codec(from_config: bool) -> None:
+    codec = PngCodec()
+    if from_config:
+        config = codec.get_config()
+        codec = PngCodec.from_config(config)
+    arr = np.ones((1000, 1000, 3), dtype="uint8")
```

Comment:
arr & _arr are not good variable names, i recommend `expected_array` & `actual_array`